eng/templates/regression-test-jobs.yml

```diff
@@ -0,0 +1,237 @@
+# Template for F# Compiler Regression Tests
+# Tests third-party F# projects with the freshly built compiler
+
+parameters:
+- name: testMatrix
+  type: object
+
+jobs:
+# Test against third-party repositories
+- ${{ each item in parameters.testMatrix }}:
+  - job: RegressionTest_${{ replace(item.repo, '/', '_') }}
```

Comment:
I have a bad feeling this is not being triggered at all.
<img width="272" height="264" alt="image" src="https://github.com/user-attachments/assets/0017080a-fa72-4d8f-8bda-3bd7bc5c78f5" />